src/core/core.scale.js

```diff
@@ -1468,6 +1468,7 @@ export default class Scale extends Element {
 		const labelSizes = me._getLabelSizes();
 		const tickAndPadding = tl + padding;
 		const widest = labelSizes.widest.width;
+		const lineSpace = labelSizes.highest.offset * 0.8;
```

Comment:
I added this to make the visual tests pass (that's where it would end up before) but not sure if it makes sense to do it this way. Maybe it's better to set a different type of padding?

Reply:
That's how its calculated in `fit`, so I guess it's ok.